app/models/concerns/client_search.rb

```diff
@@ -22,9 +22,7 @@ def self.text_searcher(text, sorted:, resolve_for_join_query: false)
 
       # should never match
       never_cond = sa[:id].eq(nil)
-
-      max_pk = 2_147_483_648 # PK is a 4 byte signed INT (2 ** ((4 * 8) - 1))
-      term_is_possibly_pk = numeric ? text.to_i < max_pk : false
+      term_is_possibly_pk = possibly_pk?(text)
```

Comment:
This concern is shared by the Warehouse and HMIS `Client` models -- so I think this will break for the warehouse search, since `possibly_pk?` won't be defined.

I can't think of a good place for this method to go for code-sharing, except maybe the [ApplicationHelper](https://github.com/greenriver/hmis-warehouse/blob/release-118/app/helpers/application_helper.rb). (@eanders thoughts on that?)

Reply:
Seems like `ApplicationHelper` is only automagically included in Rails views. There are a couple places where we explicitly `include ApplicationHelper`, but that's not quite working here -- getting the error,
```
undefined method `possibly_pk?' for #<ActiveRecord::Relation ... >
```